packages/@o3r/design/schematics/ng-add/index.ts

```diff
@@ -24,22 +24,14 @@ export function ngAddFn(options: NgAddSchematicsSchema): Rule {
   /* ng add rules */
   return async (tree) => {
     const { getPackageInstallConfig, setupDependencies, setupSchematicsParamsForProject } = await import('@o3r/schematics');
+    const defaultSchematicOptions = Object.fromEntries([
+      '@o3r/core:component',
+      '@o3r/core:component-presenter',
+      '*:*'
+    ].map((name) => [name, { useOtterDesignToken: true }]));
```

Comment:
Is it really equivalent ? We have a two entry list here when it was a key value object before?

Reply:
![image](https://github.com/user-attachments/assets/81104896-08a4-4aee-9066-f6b09da9e1fa)
Object.fromEntries return an object

Reply:
https://github.com/AmadeusITGroup/otter/pull/2384